eblocker-icapserver/src/test/java/org/eblocker/server/common/blacklist/CacheTest.java

```diff
@@ -357,26 +357,6 @@ void storeRemoveBloomOnlyFilter() throws IOException {
         assertEquals(0, pathCount("/" + LIST_DIR));
     }
 
-    @Test
-    void storeRemoveFileOnlyFilter() throws IOException {
-        //Given
-        Path filePath = createTestFile(0);
-
-        Cache cache = new Cache(cachePath, objectMapper);
-
-        //When
-        CachedFileFilter cachedFilter = cache.storeFileFilter(0, 0, "domainblacklist/string", filePath.toString(), null);
```

Comment:
If I have not made a mistake, then the bloomFilter in the production code can never be null, so this test can be deleted.